data_steward/tools/deid_runner.sh

```diff
@@ -101,8 +111,12 @@ python "${TOOLS_DIR}/run_deid.py" --idataset "${cdr_id}" --private_key "${key_fi
 # create empty sandbox dataset for the deid
 bq mk --dataset --force --description "${version} sandbox dataset to apply cleaning rules on ${registered_cdr_deid}" --label "phase:sandbox" --label "data_tier:registered" --label "release_tag:${dataset_release_tag}" --label "de_identified:true" "${APP_ID}":"${registered_cdr_deid_sandbox}"
 
+# clear GOOGLE_APPLICATION_CREDENTIALS environment variable inorder to make impersonation work
+unset GOOGLE_APPLICATION_CREDENTIALS
+gcloud config set account "${pmi_email}"
+
 # apply de-identification rules on registered tier dataset 
-python "${CLEANER_DIR}/clean_cdr.py" --project_id "${APP_ID}" --dataset_id "${registered_cdr_deid}" --sandbox_dataset_id "${registered_cdr_deid_sandbox}" --data_stage ${data_stage} --mapping_dataset_id "${cdr_id}"  --cope_lookup_dataset_id "${cope_lookup_dataset_id}" --cope_table_name "${cope_table_name}" -s 2>&1 | tee registered_tier_cleaning_log.txt
+python "${CLEANER_DIR}/clean_cdr.py" --project_id "${APP_ID}" --dataset_id "${registered_cdr_deid}" --run_as "${run_as}" --sandbox_dataset_id "${registered_cdr_deid_sandbox}" --data_stage ${data_stage} --mapping_dataset_id "${cdr_id}"  --cope_lookup_dataset_id "${cope_lookup_dataset_id}" --cope_table_name "${cope_table_name}" --deid_questionnaire_response_map_dataset "2022q2r1_rdr_sandbox" -s 2>&1 | tee registered_tier_cleaning_log.txt
```

Comment:
The `deid_questionnaire_response_map_dataset` argument is hardcoded here. Should it be a variable from a shell script command line argument?